src/screens/MoreScreen.js

```diff
@@ -43,12 +43,6 @@ class MoreScreen extends Component {
             title: 'More',
             headerTitle: 'More Options',
             headerLeft: null,
-            headerTitleStyle: {
-              color:'white'
-             },
-             headerStyle: {
-              backgroundColor: badgeGreen
-             },
```

Comment:
One of several examples where the component-specific `NavigationOptions` can be removed in favor of the global defaults we set in `App.js`